File: Example4OC/Example4OC/TestConfig/BVHUD.m
//
//  BVHUD.m
//  LibosExample
//
//  Created by 黑眸智能 on 2021/11/2.
//

#import "BVHUD.h"
#import <SVProgressHUD.h>

@implementation BVHUD

+(void)showInfo:(NSString *)info{
    [SVProgressHUD showInfoWithStatus:info];
}

+(void)showLoading:(NSString *)info {
    [SVProgressHUD showWithStatus:info];
}

+(void)dismiss {
    [SVProgressHUD dismiss];
}

@end

File: Example4Swift/Example4Swift/TestConfig/BVHUD.swift
//
//  BVHUD.swift
//  Libos
//
//  Created by 黑眸智能 on 2021/10/13.
//

import UIKit
import SVProgressHUD

class BVHUD: NSObject {
    class func showInfo(_ info: String) {
        SVProgressHUD.showInfo(withStatus: info)
    }
    
    class func showLoading(_ loading: String) {
        SVProgressHUD.show(withStatus: loading)
    }
    
    class func dismiss() {
        SVProgressHUD.dismiss()
    }
}
